docs/documentation/getting-started/load.mdx

```diff
@@ -2,19 +2,29 @@
 title: Load Data from Postgres
 ---
 
-The easiest way to copy data from another Postgres into ParadeDB is with the `pg_dump` and `pg_restore` utilities. These are installed by default when you install `psql`.
+The easiest way to copy data from another Postgres into ParadeDB is with the `pg_dump` and `pg_restore` utilities. These are
+installed by default when you install `psql`.
 
-This approach is ideal for quickly bootstrapping your ParadeDB cluster with some data. To keep ParadeDB in sync with an existing Postgres database in real time, [logical replication](/deploy/replication) should be configured.
+This approach is ideal for quickly bootstrapping a development ParadeDB cluster
+with a snapshot of your data. To deploy ParadeDB in production, kept in sync with an existing Postgres database in real
+time, [logical replication](/deploy/replication) should be configured instead.
+
+<Note>
```

Comment:
This section is for "loading data." This info should all be put into the deployment section.

Reply:
Good point, done